src/strategies/multichain/index.ts

```diff
@@ -62,6 +63,43 @@ async function getChainBlocks(
   return chainBlocks;
 }
 
+async function getChainBlocksFromApi(
+  snapshot,
+  provider,
+  options
+): Promise<any> {
```

Comment:
As long as `resp.blocks` isn't a promise.... the return value here is an object I guess.